src/components/Dashboard/TasksColumn/TasksColumn.jsx

```diff
@@ -26,14 +26,14 @@ class TasksColumn extends Component {
 
   render() {
     const {
-      title, tasks, loading, allSortData,
+      title, filteredTasks, loading, allSortData = {},
     } = this.props;
 
     const sortIconColor = loaderColor(title);
 
     const storageKey = `${document.URL.split('/').pop()}:${title}`;
     const sortState = allSortData[storageKey] || {};
-    let sortedTasks = tasks;
+    let sortedTasks = filteredTasks;
     if (sortState.direction !== 'NONE') {
       sortedTasks = sort(sortedTasks, sortState.field, sortState.direction);
```

Comment:
Mutating a variable and throwing it into a function as an argument isn't the best idea. Please rewrite without `let` and without this -- `sortedTasks = sort(sortedTasks`

Reply:
As you see in [sortUtils.js:2](https://github.com/Lv398NodeJS/add-a-thing/blob/67b977596c9106d35f0f370432d23310055bac93/src/components/Dashboard/SortList/sortUtils.js#L2), we are not mutating array.

Reply:
Change it to `const` and see if it works

Reply:
sortedTasks may sorted inside himself, how works native .sort method.
<img width="961" alt="Screenshot 2019-05-31 at 12 18 50 AM" src="https://user-images.githubusercontent.com/16022500/58665611-f1865d00-8339-11e9-99c2-498dd118069a.png">

Reply:
Also, u can use mapper function for sort

sortedTasks.sort(yourMapper(field, direction))

const yourMapper = (field, direction) => (a, b) => { // code mapper }

Reply:
You are updating a variable to a result of a function where this variable is an argument. Does this sound in any way okay to you?